File: template-java-spark/src/main/java/org/codecop/rogue/template/Hello.java
package org.codecop.rogue.template;

/**
 * Domain class with JSON support.
 */
public class Hello {

    private String name = "World!";

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String nameAsJson() {
        return "{ \"name\": \"" + getName() + "\"}";
    }

}

File: jj/hello/hello.go
package hello

type Hello struct {
	name string
}

func NewHello(name string) Hello {
	return Hello{name: name}
}

func (h *Hello) GetName() string {
	return h.name
}

func (h *Hello) SetName(name string) {
	h.name = name
}
